appinventor/appengine/src/com/google/appinventor/client/editor/youngandroid/YaBlocksEditor.java

```diff
@@ -180,6 +181,17 @@ public DropTarget[] getDropTargets() {
 
   // FileEditor methods
 
+  @Override
+  public DropTargetProvider getDropTargetProvider() {
```

Comment:
IntelliJ claims that the abstract definition of this method is never used. We should consider removing the dead code.

Reply:
> IntelliJ claims that the abstract definition of this method is never used. We should consider removing the dead code.

This method is required by the abstract class. I can make it empty, but it seems like the right thing to do would be to look at the abstract class.